resolver/caching_resolver.go

```diff
@@ -66,6 +76,19 @@ func configurePrefetching(c *CachingResolver, cfg *config.CachingConfig) {
 	})
 }
 
+func setupRedisSubscribers(c *CachingResolver) {
+	logger := logger("caching_resolver_redis")
+
+	go func() {
+		for rc := range c.redisClient.Channel {
+			if rc != nil {
+				logger.Debug("Key: ", rc.Key, " Value: ", rc.Response)
+				c.putInCache(rc.Key, rc.Response, false)
+			}
+		}
+	}()
+}
+
 // check if domain was queried > threshold in the time window
 func (r *CachingResolver) isPrefetchingDomain(cacheKey string) bool {
```

Comment:
I'm thinking about prefetching: if we do not publish cache reloads from prefetching -> caches on different blocky instances will be different. But on the other side: each blocky instance will prefetch different domains.
What do you think?